core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -3156,6 +3156,39 @@ public static Object regexpMatchCount(Object str, Object regex, Object startPos,
     }
     return count;
   }
+
+  public static Object cotFunction(Double value) {
+    return 1 / Math.tan(value);
+  }
+
+  public static Object bitwiseAnd(Integer value1, Integer value2) {
```

Comment:
value1 can be re-write as firstOperand same as value2.
Revisit all the variable declarations.